test/fixtures/v2/documentDnsDidNoDns.ts

```diff
@@ -0,0 +1,55 @@
+export const documentDnsDidNoDns: any = {
```

Comment:
from the test file name I actually expected the location key to not be present, shall we rename documentDnsDidNoDnsTXTEntry or sth similar ?